run_test.sh

```diff
@@ -308,7 +309,7 @@ run_with_expected_output dkms build -k "${KERNEL_VER}" -m dkms_test -v 1.0 << EO
 
 Building module:
 Cleaning build area...
-make -j$(nproc) KERNELRELEASE=${KERNEL_VER} -C /lib/modules/${KERNEL_VER}/build M=/var/lib/dkms/dkms_test/1.0/build...
+make -j${nproc} KERNELRELEASE=${KERNEL_VER} -C /lib/modules/${KERNEL_VER}/build M=/var/lib/dkms/dkms_test/1.0/build...
```

Comment:
as parallelism is not really beneficial for the tests, we could just run them under `parallel_jobs=1` and `s/\$\(nproc\)/1/`

Reply:
replaced the patch with the parallel_jobs=1 approach

Reply:
Haven't looked all the corner-cases but I believe you're correct.

Then again, such change should really be a separate PR. A cleaner option would be to `export MAKEFLAGS="-j..."` at the top, instead of repeatedly passing it to each make invocation.

Reply:
Split and addressed in separate PR.